src/screens/SettingsScreen.jsx

```diff
@@ -1,23 +1,39 @@
-import React from 'react';
+import React, { useContext } from 'react'
 import { StyleSheet, Text, View } from 'react-native';
 import ExampleComponent from '../components/ExampleComponent';
+import UserContext from '../contexts/UserContext';
+import userlogo from '../images/userlogo.png';
 
 const SettingsScreen = () => {
+    const { user } = useContext(UserContext);
+    
     return (
         <View style={styles.container}>
-            <Text>Example screen for settings - hello world!</Text>
+            <Text>Example screen for profile - hello world!</Text>
             <ExampleComponent />
+            <br></br>
+            <img style={{width: '100px', height: '100px',}} src={userlogo} alt="this is user image" />
```

Comment:
Instead of using an image, we could use the same icon that's used in the bottom tab bar. Check out line 20 of `App.js` to see what component to use, and where to import it from. You can use `person-circle` as the "name" property.